### DISCUSSION TITLE
Automatic score generator

### DISCUSSION THREAD

**Discussion by CharlesToulouse** [ryohey/signal#300] [Ideas]
Love the design of Signal however the main function I would need is a score generator because no one reads MIDI files.
A score generator with a few options for tweaking would be great.

**Comment by alex32125** [comment]
In my opinion, if you want to look at a score, you'd better use another software because signal is not a notation software. Piano roll notation is not meant to be read, but easily placed. If you are having trouble using the piano roll, I'd recommend practicing using it. You will understand it the more you use it

**Reply by CharlesToulouse** [reply]
It would be nice to have a score generator that's all. But you're absolutely right I can use the MIDI file inside my DAW to generate a score, it doesn't have to be integrated.
